libs/wingsdk/test/target-tf-aws/bucket.inflight.test.ts

```diff
@@ -147,3 +148,66 @@ test("delete object from a bucket with mustExist option", async () => {
   // THEN
   expect(response).toEqual(undefined);
 });
+
+test("Given a non public bucket when reaching to a key public url it should throw an error", async () => {
+  // GIVEN
+  const BUCKET_NAME = "BUCKET_NAME";
+  const KEY = "KEY";
+
+  s3Mock
+    .on(GetBucketLocationCommand, { Bucket: BUCKET_NAME })
+    .resolves({ LocationConstraint: "us-east-2" });
+
+  // WHEN
+
+  expect(async () => {
```

Comment:
If you run `npm run build` within libs/wingsdk does it build without complaining about promises?

Reply:
it did actually :) just fixed